UserApi/Controllers/UserController.cs

```diff
@@ -123,17 +140,14 @@ public async Task<ActionResult> DeleteUser(Guid userid, [FromBody] User user)
 
         /// <summary>
         /// Changes a user's permissionlevel up to an admin level's (3)
+        /// If updating permissions fails, return 404
+        /// If updating is sucessful, return 202
         /// </summary>
         /// <param name="userid"></param>
         /// <returns></returns>
         [HttpPost("addadmin/{userid}")]
-        public async Task<ActionResult> AddAsAdmin(Guid userid)
+        public async Task<ActionResult> AddAsAdmin(string userid)
```

Comment:
You should still check that the userid isn't empty, null, etc.

Reply:
The check is done in the repo layer, and will return false all the way back up to the controller, and return 404